examples/github-complete/variables.tf

```diff
@@ -22,3 +22,18 @@ variable "github_user" {
   description = "Github user for Atlantis to utilize when performing Github activities"
   type        = string
 }
+
```

Comment:
and user and token variables here

Reply:
I cannot remove this because it would break the repository webhook creation.